src/Tribe/Views/V2/Views/Day_View.php

```diff
@@ -41,30 +41,22 @@ public function prev_url( $canonical = false, array $passthru_vars = [] ) {
 		$date           = $this->context->get( 'event_date', $default_date );
 		$event_date_var = $default_date === $date ? '' : $date;
 
-		$previous_date = date( Dates::DBDATEFORMAT, strtotime( $date . '00:00 -1day' ) );
-
-		$past = tribe_events()->by_args( [
-			'on_date'  => $previous_date,
-			'posts_per_page' => 1,
-			'order'          => 'DESC',
-		] );
-		$past_posts = $past->get_query()->get_posts();
-		$previous_event = reset( $past_posts );
-
-		if ( ! $previous_event ) {
-			$past = tribe_events()->by_args( [
-				'starts_before'  => $date,
-				'posts_per_page' => 1,
-				'order'          => 'DESC',
-			] );
-			$past_posts = $past->get_query()->get_posts();
-			$previous_event = reset( $past_posts );
-
-			if ( ! $previous_event ) {
+		$previous_date = date( Dates::DBDATEFORMAT, strtotime( $date . ' -1day' ) );
```

Comment:
`date` relies on the system locale and timezone settings, those might not be the same as the site settings.
Modify this code to insulate it:

```php
$one_day = new \DateInterval( 'P1D' );
$previous_date = Tribe__Date_Utils::build_date_object( $date)->sub( $one_day );
```

At the top of the file add `use Tribe__Date_Utils as Dates` for consistency.